package-lock.json

```diff
@@ -6,7 +6,7 @@
     "packages": {
         "": {
             "name": "@mparticle/web-braze-kit",
-            "version": "4.0.1",
+            "version": "4.1.1",
```

Comment:
Please roll this back.

Reply:
Looking at the [git blame](https://github.com/mparticle-integrations/mparticle-javascript-integration-braze/blame/b043d4637a7e0b6d82bf550d63936a77d6b2e0c2/package-lock.json#L9) it says that you made the change in the master-v4 branch a few days ago. I could very well be confused about these changes.